source-release/pom.xml

```diff
@@ -141,6 +141,7 @@ under the License.
                       <value>${project.build.directory}/it-repo</value>
                     </property>
                   </systemProperties>
+                  <argLine>-Dmaven.repo.local="${project.build.directory}/it-repo"</argLine>
```

Comment:
IS this relevant to the IT? Maybe it should be separate?!

Reply:
This was a temporary test for https://issues.apache.org/jira/browse/MASFRES-70.

Reply:
system property set by failsafe should works - you don't need use argLine
By the way - we should first try to upgrade verifier to 2.0.0-M1

Reply:
It doesn't as outlined in https://issues.apache.org/jira/browse/MASFRES-70 and https://issues.apache.org/jira/browse/MASFRES-70.